Subject: Revert versioning changes
From: Michal Čihař <michal@cihar.com>

Upstream commit
https://github.com/hughsie/libgusb/commit/cfaab3e523c11800b6d77c1d10ce0c71799a4482
has changed versions for symbols which were already made public in previous
release. This patch reverts it so that the ABI doesn't change between 0.2.9 and 0.2.10.

This patch should be temporary until resolved upstream:
https://github.com/hughsie/libgusb/issues/9
Index: libgusb/gusb/libgusb.ver
===================================================================
--- libgusb.orig/gusb/libgusb.ver
+++ libgusb/gusb/libgusb.ver
@@ -42,6 +42,26 @@ LIBGUSB_0.1.0 {
     g_usb_source_error_quark;
     g_usb_source_set_callback;
     g_usb_strerror;
+# LIBGUSB_0.2.5
+    g_usb_context_get_main_context;
+    g_usb_context_set_main_context;
+# LIBGUSB_0.2.8
+    g_usb_device_get_interface;
+    g_usb_device_get_interfaces;
+    g_usb_device_get_release;
+    g_usb_device_set_interface_alt;
+    g_usb_interface_get_alternate;
+    g_usb_interface_get_class;
+    g_usb_interface_get_extra;
+    g_usb_interface_get_index;
+    g_usb_interface_get_kind;
+    g_usb_interface_get_length;
+    g_usb_interface_get_number;
+    g_usb_interface_get_protocol;
+    g_usb_interface_get_subclass;
+    g_usb_interface_get_type;
+# LIBGUSB_0.2.9
+    g_usb_context_wait_for_replug;
   local: *;
 };
 
@@ -81,43 +101,16 @@ LIBGUSB_0.2.4 {
 
 LIBGUSB_0.2.5 {
   global:
-    g_usb_context_get_main_context;
-    g_usb_context_set_main_context;
     g_usb_device_get_custom_index;
   local: *;
 } LIBGUSB_0.2.4;
 
-LIBGUSB_0.2.8 {
-  global:
-    g_usb_device_get_interface;
-    g_usb_device_get_interfaces;
-    g_usb_device_get_release;
-    g_usb_device_set_interface_alt;
-    g_usb_interface_get_alternate;
-    g_usb_interface_get_class;
-    g_usb_interface_get_extra;
-    g_usb_interface_get_index;
-    g_usb_interface_get_kind;
-    g_usb_interface_get_length;
-    g_usb_interface_get_number;
-    g_usb_interface_get_protocol;
-    g_usb_interface_get_subclass;
-    g_usb_interface_get_type;
-  local: *;
-} LIBGUSB_0.2.5;
-
-LIBGUSB_0.2.9 {
-  global:
-    g_usb_context_wait_for_replug;
-  local: *;
-} LIBGUSB_0.2.8;
-
 LIBGUSB_0.2.11 {
   global:
     g_usb_context_get_flags;
     g_usb_context_set_flags;
   local: *;
-} LIBGUSB_0.2.9;
+} LIBGUSB_0.2.5;
 
 LIBGUSB_0.3.1 {
   global:
